src/components/ProductCard.tsx

```diff
@@ -156,7 +102,7 @@ export interface IProps {
 
 const ProductCard: React.FC<IProps> = ({
```

Comment:
There is a TODO regarding adding loading state. What is the desired loading visual?

Reply:
Hmm not sure 😆 wouldn't worry too much since that comment is 20 months old and probably we are going to refactor this with the DS.